zdm/real_loading.py

```diff
@@ -114,14 +118,15 @@ def surveys_and_grids(init_state=None, alpha_method=1, add_20220610A=False):
 
     # get the grid of p(DM|z)
     zDMgrid, zvals,dmvals = misc_functions.get_zdm_grid(
-        state, new=True, plot=False, method='analytic', nz=5000,
+        state, new=True, plot=False, method='analytic', nz=2000, ndm=5600,
```

Comment:
what is the right size for these?

Reply:
I believe that the `nz=5000` was the right number of `z` bins from Esan's analysis, but we ran into memory issues with sampling such a large grid so we decided to run `nz=2000` only until the final cube for the paper (added this to the `TODO.txt`).

Additionally, we said that `ndm=5600` was appropriate since the default bin count is `ndm=1400`.